tasks/main.yml

```diff
@@ -1,2 +1,21 @@
 ---
-# tasks file for skeleton
+# tasks file for apt-over-https
+
+- name: Perform any setup before altering apt repository URLs
+  ansible.builtin.include_tasks: setup.yml
+
+- name: Load tasks file with install tasks based on the OS type
+  ansible.builtin.include_tasks: "{{ lookup('first_found', params) }}"
+  vars:
+    params:
+      files:
+        - "{{ ansible_distribution }}_{{ ansible_distribution_release }}.yml"
+        - "{{ ansible_distribution }}.yml"
+        - "{{ ansible_os_family }}.yml"
+      paths:
+        - "{{ role_path }}/tasks"
+
+- name: Update the cache using the updated sources
+  ansible.builtin.apt:
+    force_apt_get: yes
```

Comment:
Why are you including `force_apt_get`?  Why not just install `aptitude` in your `tasks/setup.yml`?  We do something similar everywhere else anyway, so `aptitude` is going to be installed eventually.

Reply:
Do you remember which project that is, so I can refresh my memory?

Reply:
It's used in [cisagov/ansible-role-docker](https://github.com/cisagov/ansible-role-docker/blob/8bdbfc4bd10882767e613069dfbd2d23c16306e3/tasks/setup_Debian.yml#L22-L28) actually.

Reply:
Please see [ddf95bc](https://github.com/cisagov/ansible-role-apt-over-https/pull/3/commits/ddf95bc68c8b28790cf640537d930e161ba59e27).

Reply:
Please see [here](https://github.com/cisagov/ansible-role-apt-over-https/pull/3#discussion_r855366765).

Reply:
Updated in [a9c6320](https://github.com/cisagov/ansible-role-apt-over-https/pull/3/commits/a9c63208770d0a72c5b6b836d34b0c250c36bd14).